src/NuGet.Clients/PackageManagement.UI/Xamls/ProjectView.xaml

```diff
@@ -78,7 +78,7 @@
       MinHeight="24"
       AutomationProperties.AutomationId="Project_Button_Uninstall"
       HorizontalAlignment="Left"
-      Visibility="{Binding InstalledVersion, Converter={StaticResource NullToVisibilityConverter}, Mode=OneWay}"
```

Comment:
maybe it could have both thinking about it now. I think for installed packages that cannot be uninstalled disabled is more clear.